packages/client/tests/functional/_utils/setupQueryEngine.ts

```diff
@@ -14,18 +14,17 @@ import path from 'path'
  * @param platform
  */
 export async function setupQueryEngine(clientEngineType: ClientEngineType, platform: Platform) {
```

Comment:
This is commented out, for the time being, you will need to delete the current version and then copy your built version of the engine into the `./packages/client` dir.